README.md

```diff
@@ -1,24 +1,76 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# The Alleyway
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [File Structure](#file-structure)
+3. [Technologies](#technologies)<br />
+   - [Prismic](#prismic)
+4. [Developers](#developers)
 
 ## Getting Started
 
-First, run the development server:
+1. **Clone the repository:**
+
+```bash
+git clone https://github.com/Scout-NU/alejandra-site.git
+```
+
+2. **Change directories**
+
+```bash
+cd alejandra-site
+```
+
+3. **Set up environment variables**
+   In the project directory there is a file called `env_example`, which holds all the necessary environment variables. Most importantly, it contains the Prismic access token.
+
+Copy the file and add the values to the variables.
+
+```sh
+cp env_example .env
+```
+
+```sh
+PRISMIC_URL="XXXXXXXXXXXXXXX"
+```
+
+4. **Install the packages**
+
+```bash
+# In the project directory
+npm install
+```
+
+5. **Run the server locally**
 
 ```bash
 npm run dev
```

Comment:
Would recommend yarn instead of npm because it can be more reliable for various things, but up to you.